include/eve/detail/function/simd/common/to_logical.hpp

```diff
@@ -58,7 +59,10 @@ EVE_FORCEINLINE auto
 to_logical(C c, eve::as<T>) noexcept
 {
   using l_t = typename as_logical<T>::type;
-  using i_t = as_integer_t<typename as_logical_t<T>::mask_type>;
+  using i_t = std::conditional_t< (T::size()>=128 && sizeof(element_type_t<T>) == 1)
+                                , as_integer_t<typename as_logical_t<T>::mask_type,unsigned>
+                                , as_integer_t<typename as_logical_t<T>::mask_type>
+                                >;
```

Comment:
What's happening? 256 chars overflow?
If unsigned integer works - why not always use unsigned?

Reply:
x86 unsigned integer comparisons are more costly than signed ones.
This prevents pessimisation.